stack.go

```diff
@@ -0,0 +1,21 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+)
+
+func Stack() slog.Attr {
+	var stack string
+	pcs := make([]uintptr, 10)
+	frames := runtime.CallersFrames(pcs[:runtime.Callers(2, pcs)])
+	for {
+		frame, more := frames.Next()
+		stack += fmt.Sprintf("%s\n", frame.Function)
+		if !more {
+			break
+		}
+	}
+	return slog.Attr{Key: "stack", Value: slog.StringValue(stack[:len(stack)-1])}
```

Comment:
I think we should create const for all hardcoded Key names used in a package - to make it easier for someone to implement ReplaceAttr func to rename such keys if needed. Name it e.g. `const KeyStack = "stack"`.

Reply:
Probably even better implementation should delay stack generation until needed (by returning LogValuer instead of already resolved slog.Attr), but this will require calling runtime.Callers now and then re-implementing runtime.Stack logic inside LogValue method, which may be too complicated for initial version of this feature and can be done later (but in this case it may worth to create an issue for this, at least).